### ISSUE TITLE
Define logic within 'session_model_generate.py'

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#1072]
We need to define [`session_model_generate.py`](https://github.com/jeff1evesque/machine-learning/blob/5f0e65cafa1d4eb04a9db1e6d9b6b0321e3fb408/python/session_model_generate.py), such that a `class` structure is emphasized in [accordance](https://github.com/jeff1evesque/machine-learning/blob/a5cea9561652f6d31e73916d2977e58954b14662/python/load_logic.py#L54) to earlier [changes](https://github.com/jeff1evesque/machine-learning/issues/1051) within `load_logic.py`.

**Comment by jeff1evesque** [comment]
After #1249 has been resolved, we need to invoke the respective class method to return the correspond dataset associated with the selected stored session.

**Comment by jeff1evesque** [comment]
We will explicitly inherit the new_style class within the Model_Generate class.

**Comment by jeff1evesque** [comment]
If we replace the `return db_return['result']` with `print db_return['result']`, we get the following output within Chromes Developer Console:

```
"JSON object from Server: [
  "[\"(('dep-variable-1', 'indep-variable-1', 23.45), ('dep-variable-1', 'indep-variable-2', 98.01), ('dep-variable-1', 'indep-variable-4', 325.0), ('dep-variable-1', 'indep-variable-5', 54.64), ('dep-variable-1', 'indep-variable-6', 0.002), ('dep-variable-1', 'indep-variable-7', 23.0), ('dep-variable-1', 'indep-variable-3', 0.432), ('dep-variable-4', 'indep-variable-1', 22.1), ('dep-variable-4', 'indep-variable-2', 95.96), ('dep-variable-4', 'indep-variable-4', 342.0), ('dep-variable-4', 'indep-variable-5', 66.67), ('dep-variable-4', 'indep-variable-6', 0.001), ('dep-variable-4', 'indep-variable-7', 32.0), ('dep-variable-4', 'indep-variable-3', 0.743), ('dep-variable-5', 'indep-variable-1', 23.27), ('dep-variable-5', 'indep-variable-2', 95.03), ('dep-variable-5', 'indep-variable-4', 295.0), ('dep-variable-5', 'indep-variable-5', 55.83), ('dep-variable-5', 'indep-variable-6', 0.001), ('dep-variable-5', 'indep-variable-7', 27.0), ('dep-variable-5', 'indep-variable-3', 0.488), ('dep-variable-3', 'indep-variable-1', 22.67), ('dep-variable-3', 'indep-variable-2', 101.21), ('dep-variable-3', 'indep-variable-4', 427.0), ('dep-variable-3', 'indep-variable-5', 75.45), ('dep-variable-3', 'indep-variable-6', 0.002), ('dep-variable-3', 'indep-variable-7', 26.0), ('dep-variable-3', 'indep-variable-3', 0.832), ('dep-variable-5', 'indep-variable-1', 19.99), ('dep-variable-5', 'indep-variable-2', 97.78), ('dep-variable-5', 'indep-variable-4', 303.0), ('dep-variable-5', 'indep-variable-5', 58.88), ('dep-variable-5', 'indep-variable-6', 0.001), ('dep-variable-5', 'indep-variable-7', 29.0), ('dep-variable-5', 'indep-variable-3', 0.638), ('dep-variable-5', 'indep-variable-1', 23.27), ('dep-variable-5', 'indep-variable-2', 95.03), ('dep-variable-5', 'indep-variable-4', 295.0), ('dep-variable-5', 'indep-variable-5', 55.83), ('dep-variable-5', 'indep-variable-6', 0.001), ('dep-variable-5', 'indep-variable-7', 27.0), ('dep-variable-5', 'indep-variable-3', 0.488), ('dep-variable-4', 'indep-variable-1', 20.71), ('dep-variable-4', 'indep-variable-2', 99.33), ('dep-variable-4', 'indep-variable-4', 342.0), ('dep-variable-4', 'indep-variable-5', 75.67), ('dep-variable-4', 'indep-variable-6', 0.001), ('dep-variable-4', 'indep-variable-7', 30.0), ('dep-variable-4', 'indep-variable-3', 0.648))\",\"{\\\"status\\\": true, \\\"error\\\": null}\"]"
]" ajax_data.min.js:1
reflow: 0.28ms
```

**Comment by jeff1evesque** [comment]
We need to adjust the docblock for the `format_dataset` method, so that it corresponds to vocabulary defined by the *scikit-learn* library.  Specifically, we need to adjust our docblock to align with our logic intention to [implement](https://github.com/jeff1evesque/machine-learning/issues/1#issuecomment-73443742) scikit-learn.

**Note:** the following terms are equivalent

- features: or synonymous to our understanding of *independent variables*
- observations: or synonymous to our understanding of *dependent variables*.

**Comment by jeff1evesque** [comment]
The following will convert our dataset tuple (from database) to a list:

```python
data_tuple = (('dep-variable-1', 'indep-variable-1', 23.45), ('dep-variable-1', 'indep-variable-2', 98.01),
 ('dep-variable-1', 'indep-variable-3', 0.432), ('dep-variable-1', 'indep-variable-4', 325.0),
 ('dep-variable-1', 'indep-variable-5', 54.64), ('dep-variable-1', 'indep-variable-6', 0.002),
 ('dep-variable-1', 'indep-variable-7', 25.0), ('dep-variable-2', 'indep-variable-1', 24.32),
 ('dep-variable-2', 'indep-variable-2', 92.22), ('dep-variable-2', 'indep-variable-3', 0.356),
 ('dep-variable-2', 'indep-variable-4', 235.0), ('dep-variable-2', 'indep-variable-5', 64.45),
 ('dep-variable-2', 'indep-variable-6', 0.001), ('dep-variable-2', 'indep-variable-7', 31.0),
 ('dep-variable-3', 'indep-variable-1', 22.67), ('dep-variable-3', 'indep-variable-2', 101.21),
 ('dep-variable-3', 'indep-variable-3', 0.832), ('dep-variable-3', 'indep-variable-4', 427.0),
 ('dep-variable-3', 'indep-variable-5', 75.45), ('dep-variable-3', 'indep-variable-6', 0.002),
 ('dep-variable-3', 'indep-variable-7', 24.0))
data_list = [list(row) for row in data_tuple]
```

**Comment by jeff1evesque** [comment]
Since we will [implementing](https://github.com/jeff1evesque/machine-learning/issues/1#issuecomment-73443742) `numpy`, we can safely remove our intended `format_dataset`.

**Comment by jeff1evesque** [comment]
cdd89d6: since we will [implementing](https://github.com/jeff1evesque/machine-learning/issues/1#issuecomment-73443742) `numpy`:

```python
$ python
>>> dataset = (('dep-variable-1', 'indep-variable-1', 23.45), ('dep-variable-1', 'indep-variable-2', 98.01),
 ('dep-variable-1', 'indep-variable-3', 0.432), ('dep-variable-1', 'indep-variable-4', 325.0),
 ('dep-variable-1', 'indep-variable-5', 54.64), ('dep-variable-1', 'indep-variable-6', 0.002),
 ('dep-variable-1', 'indep-variable-7', 25.0), ('dep-variable-2', 'indep-variable-1', 24.32),
 ('dep-variable-2', 'indep-variable-2', 92.22), ('dep-variable-2', 'indep-variable-3', 0.356),
 ('dep-variable-2', 'indep-variable-4', 235.0), ('dep-variable-2', 'indep-variable-5', 64.45),
 ('dep-variable-2', 'indep-variable-6', 0.001), ('dep-variable-2', 'indep-variable-7', 31.0),
 ('dep-variable-3', 'indep-variable-1', 22.67), ('dep-variable-3', 'indep-variable-2', 101.21),
 ('dep-variable-3', 'indep-variable-3', 0.832), ('dep-variable-3', 'indep-variable-4', 427.0),
 ('dep-variable-3', 'indep-variable-5', 75.45), ('dep-variable-3', 'indep-variable-6', 0.002),
 ('dep-variable-3', 'indep-variable-7', 24.0))
>>> arr = numpy.asarray(dataset)
>>> X = arr[:, 1:]
>>> X
array([['indep-variable-1', '23.45'],
       ['indep-variable-2', '98.01'],
       ['indep-variable-3', '0.432'],
       ['indep-variable-4', '325.0'],
       ['indep-variable-5', '54.64'],
       ['indep-variable-6', '0.002'],
       ['indep-variable-7', '25.0'],
       ['indep-variable-1', '24.32'],
       ['indep-variable-2', '92.22'],
       ['indep-variable-3', '0.356'],
       ['indep-variable-4', '235.0'],
       ['indep-variable-5', '64.45'],
       ['indep-variable-6', '0.001'],
       ['indep-variable-7', '31.0'],
       ['indep-variable-1', '22.67'],
       ['indep-variable-2', '101.21'],
       ['indep-variable-3', '0.832'],
       ['indep-variable-4', '427.0'],
       ['indep-variable-5', '75.45'],
       ['indep-variable-6', '0.002'],
       ['indep-variable-7', '24.0']], 
      dtype='|S16')
>>> Y = arr[:, 0]
>>> Y
array(['dep-variable-1', 'dep-variable-1', 'dep-variable-1',
       'dep-variable-1', 'dep-variable-1', 'dep-variable-1',
       'dep-variable-1', 'dep-variable-2', 'dep-variable-2',
       'dep-variable-2', 'dep-variable-2', 'dep-variable-2',
       'dep-variable-2', 'dep-variable-2', 'dep-variable-3',
       'dep-variable-3', 'dep-variable-3', 'dep-variable-3',
       'dep-variable-3', 'dep-variable-3', 'dep-variable-3'], 
      dtype='|S16')
```

therefore, we can safely remove our intended `format_dataset`.

**Comment by jeff1evesque** [comment]
Suppose observations (dependent variables) of the same subject could happen more than once, then, each observation (dependent variable) that had other instances, would be merged as one observation, and would have `n+m` features (independent variables), whereas an observation that occurred just once, would correctly have `n` features (independent variables).

**Comment by jeff1evesque** [comment]
This issue is a duplicate, and will be resolved within #726.  So, we have reverted the above incorrect / partial duplicated commits

**Comment by jeff1evesque** [comment]
We need to remove the superclass `Session_Base`.
